servers/dart_server.dart

```diff
@@ -6,16 +6,19 @@ const String _HOST = '0.0.0.0';
 const String _GREET = 'Hello World';
 
 _startServer(arg) async {
-  var server = await HttpServer.bind(_HOST, 9292, shared: true);
-  await for (HttpRequest request in server) {
-    request.response
+  print('Start server');
+  final server = await HttpServer.bind(_HOST, 9292, shared: true);
+  server.listen((HttpRequest request) {
+      request.response
       ..write(_GREET)
       ..close();
-  }
+  });
 }
 
-void main() {
-  for (int i = 0; i < Platform.numberOfProcessors; i++)
+void main(List<String> args) {
+  final threads = (args.length>0?int.parse(args[0]):Platform.numberOfProcessors);
+  print('CPU: ${Platform.numberOfProcessors}, Threads: ${threads}');
+  for (int i = 1; i < threads; i++)
```

Comment:
@costajob pay attention to 'i = 1'

Reply:
I do not get you: why starting from 1? 
You are simply reducing the number of isolates. It seems Dart runtime uses 3 threads by default, then spawn other 8, for a total of 11.
Starting by 1 does not bring any benefit on performance and it is rather obscure.